### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Suenodivino**
This is my first push to the repo.

### CODE REVIEW

**Review by YusufBolden — CHANGES_REQUESTED** [review]
Great job on setting this up. Now I want you to get practice on making changes based on code review. Add a few more lines to your README.md file

File: Dario/README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
-# 02_git_test_repo
 repo_exercise
-On my way to the finish line.
\ No newline at end of file
+On my way to the finish line.
+
+I forgot to make a new branch before pushing.
+This is the new line you asked me to add to this file.
```
Comment (Suenodivino) [inline Dario/README.md:5]: @YusufBolden This is my new line

**Review by Suenodivino — COMMENTED** [review]
@YusufBolden Is this the change you requested in my file?

**Review by YusufBolden — APPROVED** [review]
Nice work

**Pull request merged**